src/app/home/navigation.component.ts

```diff
@@ -29,6 +44,26 @@ export class NavigationComponent {
     { link: '', name: 'Feedback' },
   ];
 
+  languages = [
+    { text: 'English', value: 'en', icon: '../../assets/flags/us.png' },

```

Comment:
we are using languages array so instead of setting rtl or ltr on condition we can just use it from this array. so there will not be need of condition. Also in future there could be many languages so it need to adjust that as well

Reply:
ok I got it .you mean rtl and ltr should be used as a json value .